installer/manifests/keptn/charts/control-plane/values.yaml

```diff
@@ -130,6 +130,7 @@ bridge:
     userIdentifier: ""
     mongoConnectionString: ""
   nodeSelector: {}
+  automaticProvisioningMsg: ""
```

Comment:
maybe it would make sense to put this value next to `automaticProvisioningURL` under `features` for consistency?

Reply:
I think we should follow up with a separate enhancement ticket to merge under the same object `automaticProvisioning` the `serivceURL` and the `message`. Similarly to:
```

features:
  automaticProvisioning:
    serviceURL: ""
    message: ""

```